common/genes/02_genes_accessories_hairstyles.txt

```diff
@@ -518,5 +518,43 @@ accessory_genes = {
 			boy = male
 			girl = female
 		}
+		goblin_hairstyles = {
+			index = 19
+			male = {
+				1 = male_hair_sub_saharan_01 #Short Afro
+				1 = male_hair_western_08 #Medium Wavy
+				1 = male_hair_mena_03 #Large Curly
+				1 = male_hair_mena_01 #Curly
+				1 = male_hair_mena_02 #Medium Curls
+				1 = male_hair_mena_04 #Curly Chevelure
+				1 = male_hair_byzantine_01 #Curly Bowl
+				1 = male_hair_byzantine_02 #Roman Fade
+				1 = male_hair_indian_02 #Pulled Back Medium Wavy
+				1 = male_hair_indian_03 #High Bun
+				1 = male_hair_northern_01 #Very Long Wild
+				1 = male_hair_northern_02 #Long Wild
+				1 = male_hair_northern_02
+				1 = male_hair_steppe_01 #Hair Loops
+				1 = male_hair_steppe_02 #Steppe Rider
+			}		   
+			female = {
+				1 = female_hair_western_03 #Bun
+				1 = female_hair_western_04 #Very Long Double Braids
+				1 = female_hair_western_05 #Medium Wavy
+				1 = female_hair_western_07 #Fish Tails
+				1 = female_hair_western_08 #Double Buns
```

Comment:
I feel like it's too royal and Medieval for the goblins.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/111081845-b3622980-851e-11eb-83dc-59ce04ac6bde.png)

</details>

Reply:
Removed in bf8de12a3c69ce1f80cfda13805fdb11da568491.